config/config.go

```diff
@@ -497,6 +499,21 @@ func New(v *viper.Viper) (*Configuration, error) {
 	}
 	c.setDerivedDefaults()
 
+	if err := c.RequestValidation.Parse(); err != nil {
+		return nil, err
+	}
+
+	if err := isValidCookieSize(c.HostCookie.MaxCookieSizeBytes); err != nil {
+		glog.Fatal(fmt.Printf("Max cookie size %d cannot be less than %d \n", c.HostCookie.MaxCookieSizeBytes, MIN_COOKIE_SIZE_BYTES))
+		return nil, err
+	}
+
+	glog.Info("Logging the resolved configuration:")
```

Comment:
Why did you move this logging above the map generation? This way the generated maps will not be included in the logs.

Reply:
The logging call looks to correlate to the validate call. Such that, if validation fails you can refer to the logged configuration to figure out what went wrong. The hash sets don't affect validation, they're just an optimization. so I moved them below the validation call.

Reply:
I switched it back.